databricks/koalas/base.py

```diff
@@ -579,8 +579,18 @@ def rmod(left, right):
 
         return column_op(rmod)(self, other)
 
-    __pow__ = column_op(Column.__pow__)
-    __rpow__ = column_op(Column.__rpow__)
+    def __pow__(self, other) -> Union["Series", "Index"]:
+        def pow_func(left, right):
+            return F.when(F.lit(right is np.nan) & (left == 1), left).otherwise(pow(left, right))
```

Comment:
We should use `Column.__pow__(left, right)` or `left.__pow__(right)` instead of builtin `pow`?